staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -202,8 +202,8 @@ func (f *FieldManager) UpdateNoErrors(liveObj, newObj runtime.Object, manager st
 	if err != nil {
 		atMostEverySecond.Do(func() {
 			ns, name := "unknown", "unknown"
-			accessor, err := meta.Accessor(newObj)
-			if err == nil {
+			accessor, accessorErr := meta.Accessor(newObj)
+			if accessorErr == nil {
 				ns = accessor.GetNamespace()
 				name = accessor.GetName()
 			}
```

Comment:
Wouldn't it be cuter to do:
```suggestion
			if accessor, err := meta.Accessor(newObj); err == nil {
				ns = accessor.GetNamespace()
				name = accessor.GetName()
			}
```

Reply:
+1

Reply:
Updated, thanks